spec/support/seeders.rb

```diff
@@ -48,7 +54,7 @@ def add_needed_seeders_for(seeder_class, needed_acc)
   shared_let(:needed_seeders_keys) do
     # warning: there is one limitation: RoleSeeder will only pick 'roles' key,
     # and not 'modules_permissions' key.
-    needed_seeders.map { _1.try(:seed_data_model_key) }.compact
```

Comment:
I do not like introducing exceptions. It can then evolve to accidental complexity. The idea was to keep a 1-1 relation between the key name in the seed files and the class name of the record. This way I do not have to think too much to lookup the right key given a model name.

Instead of having `roles.project`, `roles.global`, and `roles.work_package` , why not use `project_roles`, `global_roles`, and `work_package_roles` directly? This way the role seeders would work as any other seeder.

What do you think about it?